content/rs/installing-upgrading/install/plan-deployment/persistent-ephemeral-storage.md

```diff
@@ -40,8 +40,7 @@ for the persistent volume.
 
 ## Ephemeral storage
 
-Ephemeral storage is optional. If configured, the cluster stores temporary information
-that does not need to persist in the ephemeral storage.
+Ephemeral storage is optional. If configured, the cluster stores in the ephemeral storage any temporary information that does not need to be persisted.
```

Comment:
Word order suggestion
```suggestion
Ephemeral storage is optional. If configured, the cluster stores temporary information that does not need to be persisted in the ephemeral storage.
```

Reply:
The suggestion puts the phrasing back to what it was and changes the meaning. 

In the original version, "in the ephemeral storage" might be mistaken for modifying persisted.  

We could try this?

Ephemeral storage is optional. If configured, the cluster stores - in the ephemeral storage - temporary information that does not need to be persisted.

Or (but it changes it to a more passive tense - way clearer though)

Ephemeral storage is optional. If configured, temporary information that does not need to be persisted is stored by the cluster in the ephemeral storage.